admin-tools/lib/src/main/scala/com/gu/mediaservice/ImageDataMerger.scala

```diff
@@ -147,42 +157,44 @@ class ImageDataMerger(config: ImageDataMergerConfig) extends LazyLogging {
     }
   }
 
-  private def getMergedImageDataInternal(mediaId: String)(implicit ec: ExecutionContext): Future[Option[Image]] = {
-    val maybeImage: Option[Image] = gridClient.getImageLoaderProjection(mediaId, imageLoaderEndpoint)
-    maybeImage match {
-      case Some(img) =>
-        aggregate(img).map { aggImg =>
-          Some(ImageProjectionOverrides.overrideSelectedFields(aggImg))
-        }
-      case None => Future(None)
-    }
+  private def getMergedImageDataInternal(mediaId: String)(implicit ec: ExecutionContext): Future[Option[Image]] = for {
+    maybeImage <- gridClient.getImageLoaderProjection(mediaId, imageLoaderEndpoint, None)
```

Comment:
as this isn't a "full" image, do we want to call it a maybeStubImage?

Reply:
Reluctantly agreed :)